webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -1578,38 +1540,6 @@ def mocked_publish(**params):
         assert "Please wait..." in content
         assert "Processing this crash report only takes a few seconds" in content
 
-    def test_report_index_with_invalid_date_processed(self):
```

Comment:
I didn't understand what this was testing, so I nixed it. `date_processed` is derived from `submitted_timestamp` which is set by the collector so... why wouldn't it be valid?